doc/ovs-hw-offload.md

```diff
@@ -9,63 +9,148 @@ host net-device. The VF representor plays the same role as TAP devices
 in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
 arrives to the VF, and a packet sent through the VF is received by its representor.
 
+OVS Hardware Offloading requires NIC to be configured in `switchdev` mode.
+
+The operator can automate the creation and configuration of OVS bridges when the "manageSoftwareBridges" function is activated.
+
 ## Supported Ethernet controllers
 
 The following manufacturers are known to work:
 
 - Mellanox ConnectX-5 and above
 
-## Instructions for Mellanox ConnectX-5
-
 ## Prerequisites
 
 - OpenVswitch installed
-- Network Manager installed
+
+### Activate "manageSoftwareBridges" featureGate
+
+```
+kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "manageSoftwareBridges": true  } } }' --type='merge'
+```
+
 
 ### Deploy SriovNetworkNodePolicy
 
+The example policy below selects all NVIDIA ConnectX-6 Dx devices on all worker nodes.
+
+The following actions will apply to selected NICs:
+
+* set NIC's eswitch mode to `switchdev`
+
+* create 5 Virtual Functions on each Physical Function
+
+* create a separate OVS bridge (with default configuration that is suitable for HW-offloading with OVS-kernel dataplane) for each Physical Function (PF)
+
 ```yaml
 apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetworkNodePolicy
 metadata:
-  name: ovs-hw-offload
-  namespace: sriov-network-operator
+  name: connectx6dx-switchdev
```

Comment:
Do you mind changing this back to `ovs-hw-offload`? It should work with any NIC that supports switchdev and hwoffload, right? (ConnectX5, Intel E810, ...)

Reply:
done